File: src/test/java/kitchenpos/application/fixture/MenuGroupFixtures.java
package kitchenpos.application.fixture;

import java.util.UUID;
import kitchenpos.domain.MenuGroup;

public final class MenuGroupFixtures {

    private MenuGroupFixtures() {
        throw new RuntimeException("생성할 수 없는 클래스");
    }

    public static MenuGroup createMenuGroup(UUID id, String name) {
        final MenuGroup menuGroup = new MenuGroup();
        menuGroup.setId(id);
        menuGroup.setName(name);
        return menuGroup;
    }

    public static MenuGroup createMenuGroup(String name) {
        return createMenuGroup(UUID.randomUUID(), name);
    }
}

File: src/testFixtures/kotlin/domain/MenuGroupFixtures.kt
package domain

import kitchenpos.domain.MenuGroup
import java.util.UUID

object MenuGroupFixtures {
    fun makeMenuGroupOne(): MenuGroup {
        return MenuGroup().apply {
            this.id = UUID.fromString("df8d7e4f-4283-4c91-9e43-d9e2dcb6f182")
            this.name = "한마리 메뉴"
        }
    }
}
